composer.json

```diff
@@ -1,33 +1,35 @@
 {
-	"name": "silverstripe/iframe",
-	"description": "Add an iFrame pagetype to SilverStripe, to show content from another website",
-	"type": "silverstripe-module",
-	"keywords": ["silverstripe"],
-	"license": "BSD-3-Clause",
-	"authors": [
-		{
-			"name": "Mateusz Uzdowski",
-			"email": "mateusz@silverstripe.com"
-		}
-	],
-	"require": {
-		"silverstripe/framework": "^4@dev",
-		"silverstripe/cms": "^4@dev"
-	},
-	"require-dev": {
-		"phpunit/PHPUnit": "^5.7"
-	},
-	"autoload": {
+    "name": "silverstripe/iframe",
+    "description": "Add an iFrame pagetype to SilverStripe, to show content from another website",
+    "type": "silverstripe-vendormodule",
+    "keywords": ["silverstripe"],
+    "license": "BSD-3-Clause",
+    "authors": [
+        {
+            "name": "Mateusz Uzdowski",
+            "email": "mateusz@silverstripe.com"
+        }
+    ],
+    "require": {
+        "silverstripe/recipe-cms": "^1"
```

Comment:
Let's stick with the CMS as requiring the recipe involves much more than just the CMS